src/appbundle-driver.jl

```diff
@@ -0,0 +1,13 @@
+# This in an automatically generated driver script generated by JuliaHub.jl
+# when submitting an appbundle.
+let
+    path_components = [{PATH_COMPONENTS}]
+    path = abspath(pwd(), path_components...)
```

Comment:
This feels wrong, but I seems much more robust going forwards because we want to guarantee `pwd` being set inside of the appbundle dir.

Reply:
Yeah. Fortunately, it will be hidden from the user at least :slightly_smiling_face: